src/main/java/com/google/edith/servlets/UserInsights.java

```diff
@@ -33,61 +35,64 @@
 
   private String userId;
 
-  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();  
-  private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final DatastoreService datastore = 
+      DatastoreServiceFactory.getDatastoreService();  
+  private static final DateTimeFormatter dateFormatter = 
```

Comment:
static final vars should follow upper SNAKE_CASE naming convention.
i.e. `private static final DATE_TIME_FORMATTER = ...`

Reply:
Done.